src/fmu/sumo/sim2sumo/_special_treatments.py

```diff
@@ -145,11 +145,35 @@ def convert_options(options):
     return options
 
 
-def tidy(frame):
+def find_md_log(submod, options):
+    """Search options for md_log_file
+
+    Args:
+        submod (str): submodule
+        options (dict): the dictionary to check
+
+    Returns:
+        str|None: whatever contained in md_log_file
+    """
+    logger = logging.getLogger(__file__ + ".find_md_log")
+    if submod != "rft":
+        return None
+    # Special treatment of argument md_log_file
+    md_log_file = options.get("md_log_file", None)
+    try:
+        del options["md_log_file"]
+    except KeyError:
+        logger.debug("No md log provided")
```

Comment:
Is this delete necessary?

Reply:
Yes it is, because otherwise  this call would fail: `   output = extract_df(
                res2df.ResdataFiles(datafile_path),
                **kwargs,
            )` This extract_df is not "our" function but highjacked from res2df, and cannot have other arguments, this is a fix strictly for sim2sumo, but should ideally be added to res2df and deprecated here at a later stage..

Reply:
If there isn't already, could you make an issue in res2df?
I guess it's fine to handle these things in sim2sumo for now, but we should at least flag it if it is more relevant in res2df.